tests/unit_tests/test_dates.py

```diff
@@ -5,5 +5,43 @@
 
 
 class TestDates(unittest.TestCase):
+
     def test_0_type_check_dates(self):
+        """ Checks parsing of string containing a date to dtype 'date'.
+        """
         self.assertEqual(type_check_date('31/12/2010'), dtype.date)
+
+    def test_1_type_check_datetime(self):
```

Comment:
Loving the amount of tests, nice!

Reply:
Thanks! 

I do need to add a few more to test if passing seconds since epoch for dates without hours/minute/seconds really triggers type_infer to return "date" instead of "datetime". I'll implement this asap and push it.